src/io/openshift/Plugins.groovy

```diff
@@ -1,8 +1,14 @@
 package io.openshift
-import io.openshift.plugins.*
+
+import io.openshift.plugins.analytics
+import io.openshift.Globals
 
 class Plugins implements Serializable {
     static def register() {
+      Globals.plugins["analytics"] = Globals.plugins["analytics"] ?: [disabled: false]
+      if (Globals.plugins["analytics"].disabled ?: false) {
```

Comment:
isn't `Globals.plugins["analytics"].disabled` enough?

Reply:
I added this for the case if someone has added analytics but did not set `disabled` property like

```groovy

plugins analytics["loglvel" : "info"]

```